include/CppUTestExt/MockNamedValue.h

```diff
@@ -40,6 +40,7 @@ class MockNamedValueComparator
 
     virtual bool isEqual(const void* object1, const void* object2)=0;
     virtual SimpleString valueToString(const void* object)=0;
+    virtual void* copy(const void*, const void*) { return 0; }
```

Comment:
Yes, that sounds like a very good idea, especially returning a boolean and checking that the copy() method was indeed overridden. @random85 - what was the original idea behind returning a void *?